Sources/Commands/APIDigester.swift

```diff
@@ -178,7 +180,7 @@ public struct SwiftAPIDigester {
         let process = Process(
             arguments: arguments,
             outputRedirection: .none,
-            verbose: verbosity != .concise
+            verbose: TSCUtility.verbosity != .concise
```

Comment:
not sure I am reading this right... is it comparing the global value that is coming from TSCUtility? should this be self.verbosity?

Reply:
there's no `self.verbosity`, it uses `TSCUtiliry.verbosity` all the way as it used to. What we can see now is how not perfect this global is.

Reply:
right, we should do away with the global imo. but this one used to be "verbosity", where did it come from before?

Reply:
It was still from the `TSCUtility`, just used without module prefix. I made it explicit.

Reply:
okay that makes total sense